contrib/scraper_sao_luis.py

```
@@ -0,0 +1,86 @@
+import scrapy
+
+
+class SaoLuisSpider(scrapy.Spider):
+    name = "sao_luis"
+    allowed_domains = ["arquislz.org.br"]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)